app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="ui placeholder segment">
+  <h2 class="ui header center aligned">Sign Up</h2>
+  <div class="ui one column very relaxed stackable grid">
+    <div class="column">
+      <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+        <%= hidden_field_tag :token, params[:token] %>
+        <div class="field">
+          <%= f.label :email %>
+          <div class="ui left icon input">
+            <%= f.email_field :email, autofocus: true, placeholder: 'Email' %>
+            <i class="user icon"></i>
+          </div>
+        </div>
+        <div class="field">
+          <%= f.label :password %>
+          <% if @minimum_password_length %>
```

Comment:
field validations should be in JS

Reply:
@davinciRor added js validation for users sign up